webapp/channels/src/components/setting_picture.tsx

```diff
@@ -231,16 +231,12 @@ export default class SettingPicture extends Component<Props, State> {
                     >
                         {imageElement}
                     </div>
-                    <OverlayTrigger
-                        delayShow={Constants.OVERLAY_TIME_DELAY}
+                    <WithTooltip
+                        id='removeIcon'
+                        title={<div aria-hidden={true}>
+                            {title}
+                        </div>}
```

Comment:
Lets just pass the title directly instead here

Reply:
I have passed `title` directly